check_springboot_actuator.py

```diff
@@ -53,9 +54,11 @@ def request_data(url, **get_args):
     try:
         response = get(url, **get_args)
         # check response content type to determine which actuator api to be used
-        if response.ok or response.status_code == 503:
+        if response.ok:
```

Comment:
The `response.status_code == 503` has to stay. have a look on commit --> https://github.com/T-Systems-MMS/check_springboot_actuator/commit/095cc4486ae9b3a13536baf8339faec31dc92c83